argus/resources/windows/encode.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Cloudbase Solutions Srl
```

Comment:
This can be a collection of utilities. In order to prepare the CLI application you can use argparse (or the `client` module from `arestor` project).